scss/modules/_breadcrumbs.scss

```diff
@@ -0,0 +1,189 @@
+////
+/// @author       Web Team at Canonical Ltd
+/// @link         http://ubuntudesign.github.io/vanilla-framework/docs/#mixin-vf-lists
+/// @since        0.0.3
+////
+
+/// Default blockquote styling
+/// @group Blockquotes
+/// @example
+/// <blockquote>
+///   ...
+/// </blockquote>
+@mixin vf-breadcrumbs {
+  // scss-lint:disable NestingDepth
+  .nav-secondary {
+
+    @media only screen and (min-width: $breakpoint-large) {
+      border-bottom: 1px solid $alto-grey;
+    }
+
+    .breadcrumb {
+      box-sizing: border-box;
+      width: 100%;
+      margin-bottom: 0;
+      margin-left: 0;
+
+      @media only screen and (min-width: $breakpoint-large) {
+        width: auto;
+        float: left;
+        margin-left: 20px;
+      }
+
+      li {
+        display: block;
+        box-sizing: border-box;
+        width: 100%;
+        height: 40px;
+        margin: 0;
+        color: $white;
+
+        &:first-of-type {
+          border-bottom: 1px solid $alto-grey;
+
+          @media only screen and (min-width: $breakpoint-large) {
+            border-bottom: 0;
+          }
+        }
+
+        a {
+          font-size: 16px;
+          display: block;
+          box-sizing: border-box;
+          width: 100%;
+          margin-right: 0;
+          padding: 8px 10px 0;
+          text-decoration: none;
+          color: $cool-grey;
+
+          @media only screen and (min-width: $breakpoint-large) {
+            font-size: 14px;
+          }
+
+          .after {
+            position: relative;
+            top: 0;
+            right: 0;
+            float: right;
+            width: 18px;
+            height: 18px;
+            margin-top: -6px;
+            margin-right: -5px;
+            padding: 10px;
+            background-image: url('#{$asset-path}img/patterns/nav-down-arrow.svg');
+            background-repeat: no-repeat;
+            background-position: center center;
+            background-size: 18px;
+          }
+        }
+
+        > a {
```

Comment:
I wonder if we should class this up as it will be prone to errors if markup isnt exact and restricts the breadcrumbs.